src/whylogs/features/autosegmentation.py

```diff
@@ -0,0 +1,105 @@
+from typing import Dict, List, Optional, Union
+
+import numpy as np
+import pandas as pd
+
+
+def _entropy(series: pd.Series, normalized: bool = True):
```

Comment:
Can we have some unit test for these methods?

Reply:
will do